### DISCUSSION TITLE
Dropdown date picker

### DISCUSSION THREAD

**Discussion by michaeljoc** [Hopding/pdf-lib#1239] [Q&A]
Hey guys, I am wondering how to get the date picker drop down?

**Comment by dan-willett** [comment]
PDF's don't have a specific date field, you have to use a text field formatted as a date.  And apparently unless you are using a newer PDF viewer, you have to add a javascript to enable a date picker.

Here are a couple of articles on the subject:
https://community.adobe.com/t5/acrobat-discussions/how-to-add-date-picker-in-specific-forms/td-p/9050608

https://kbpdfstudio.qoppa.com/how-to-add-date-picker-on-form-fields/#:~:text=How%20to%20create%20an%20interactive%20date%20field%20with,4.%20Change%20to%20Format%20tab.%205.%20Select%20

And a date picker javascript that used to be a paid option, but is now free:
https://www.abracadabrapdf.net/utilitaires/utilities-in-english/magicdate-picker/

Looking through the source for PDF-Lib, it did not appear to me that there is any way in PDF-Lib to set the format for text field...  If somebody knows how to do this with PDF-Lib, please chime in.
